.github/workflow/go.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2025 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# GitHub Actions workflow to lint and test.
+name: "Go"
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  group: "go-${{ github.workflow }}-${{ github.event.number || github.ref }}"
+  cancel-in-progress: "${{ github.event_name == 'pull_request' }}"
+
+env:
+  GO_VERSION: "1.25.x"
+
+permissions:
+  contents: read
+
+jobs:
```

Comment:
This is missing a `Lint` job (and there is no `golangci.yml` file), although there is a `lint` target in Makefile (and it installs golangci-lint) - Is this intentional for now?